docs/installation.rst

```diff
@@ -0,0 +1,80 @@
+
+Installation
+============
+
+Downloading
++++++++++++
+
+First download the source from github. You can download a zip file ...
+::
+
+   $ wget https://github.com/rupertford/melody/archive/master.zip
+   $ unzip master.zip
+
+or clone the repository ...
+::
+ 
+   $ git clone https://github.com/rupertford/melody.git
+
+Python version
+++++++++++++++
+
+Melody is currently tested on Python 2.7.
+
+Installation
+++++++++++++
+
+Using setup.py
+--------------
+
+Melody includes an installation setup file called setup.py in its top level
+directory.
+::
+
+   $ cd <melody_dir>
+   $ python setup.py install
+
+If you do not have appropriate permissions you can perform a local
+installation instead
+::
+
+   $ cd <melody_dir>
+   $ python setup.py install --user
```

Comment:
Need to mention that, on certain systems (Ubuntu being one), the resulting installation dir is not automatically added to the PYTHONPATH.